### PR TITLE
creating Kyma example

### PR DESCRIPTION
**Pull request by obadrawi**
-Issue #106  : Creating a Kyma Mock in examples

### CODE REVIEW

File: examples/kyma-mock/test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var request = require('supertest');
+var exampleApp = require("./app.js")
+
+describe('tests openapi controllers', function () {
+    it('should work', function (done) {
+        exampleApp.then(function (app) {
+            describe('GET courses', function () {
+                it('should return 200', function (done) {
+                    request(app)
+                        .get('/kyma/courses')
```
Comment (a-thaler) [inline examples/kyma-mock/test.js]: If that test works then we did something very wrong

File: examples/kyma-mock/varkes_config.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "OpenAPI-Mock",
+    "apis": [
+        {
+            "baseurl": "/kyma",
+            "specification": "apis/kyma.yaml",
+            "name": "kyma"
+        }
+    ],
+    "request_log_path": "requests.log"
```
Comment (a-thaler) [inline examples/kyma-mock/varkes_config.json]: please remove for now the log option

File: examples/kyma-mock/apis/kyma.yaml [hunk 3]
```diff
@@ -0,0 +1,319 @@
+openapi: '3.0.0'
```
Comment (a-thaler) [inline examples/kyma-mock/apis/kyma.yaml]: That is not at all the full kyma API, please consider all three APIs located here: https://github.com/kyma-project/kyma/tree/master/docs/application-connector/docs/assets

File: examples/kyma-mock/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,9 @@
+FROM node:alpine
```
[the remark this answers is not in the document]
Comment (atakanyenel) [reply examples/kyma-mock/Dockerfile]: I aligned the docker files with #103 in a new commit.

File: examples/kyma-mock/varkes_config.json (hunk 2 above)
Comment (obadrawi) [reply examples/kyma-mock/varkes_config.json]: Done

File: examples/kyma-mock/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# openapi-mock example
```
[the remark this answers is not in the document]
Comment (obadrawi) [reply examples/kyma-mock/README.md]: Wrote a new ReadMe

File: examples/kyma-mock/test.js (hunk 1 above)
Comment (obadrawi) [reply examples/kyma-mock/test.js]: wrote tests for all apis

File: examples/kyma-mock/apis/kyma.yaml (hunk 3 above)
Comment (obadrawi) [reply examples/kyma-mock/apis/kyma.yaml]: Done

**Pull request merged**